modules/microshift-olm-deploy-ops-spec-ns.adoc

```diff
@@ -0,0 +1,185 @@
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-OLM-deploy-Operators-specific-namespace_{context}"]
+= Adding OLM-based Operators to a networked cluster in a specific namespace
+
+Use this procedure if you want to specify a single namespace for an Operator, for example, `olm-microshift`. In this example, the catalog is scoped and available in the global `openshift-marketplace` namespace. For {microshift-short} clusters that have network connectivity, Operator Lifecycle Manager (OLM) can access sources hosted on remote registries.
+
+[IMPORTANT]
+====
+All of the Operators installed in a specific namespace must have the same watch scope.
+====
+
+.Prerequisites
+* The {oc-first} is installed.
+* Operator Lifecycle Manager (OLM) is installed.
+* You have created a custom catalog that is running in the global namespace.
+
+.Procedure
+
+. Confirm that OLM is running by using the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-operator-lifecycle-manager get pod -l app=olm-operator
+----
++
+.Example output
+[source,terminal]
+----
+NAME                            READY   STATUS    RESTARTS   AGE
+olm-operator-66c97f894f-55wp2   1/1     Running   0          126m
+----
+
+. Confirm that the pod the catalog runs in is active by using the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-operator-lifecycle-manager get pod -l app=catalog-operator
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                READY   STATUS    RESTARTS   AGE
+catalog-operator-7cf77c459d-jbrd7   1/1     Running   0          126m
+----
+
+//Q which directory do we place these YAMLs in? do we need to specify?
+. Optional. Create a namespace by using the following command and example YAML:
++
+[source,terminal]
+----
+$ sudo cat _<ns.yaml>_ <1>
```

Comment:
Do we need these `cat`s? They only print the content of the file, `oc apply` is what does the job

Reply:
How do we create the YAMLS, and in what directory? Does OLM create default yamls for the user to edit?

Reply:
No, you're right, we're giving them a sample file right here.